pkg/odo/cli/dev/dev.go

```diff
@@ -126,10 +125,7 @@ func (o *DevOptions) Complete(ctx context.Context, cmdline cmdline.Cmdline, args
 		return err
 	}
 
-	o.variables, err = vars.GetVariables(o.clientset.FS, o.varFileFlag, o.varsFlag, os.LookupEnv)
-	if err != nil {
-		return err
-	}
+	o.variables = fcontext.GetVariables(ctx)
```

Comment:
Just curious as to why `pkg/odo/cli/deploy/deploy.go` was not refactored too.. Was it forgotten or is something preventing us from having the same logic in `odo deploy`? It also supports `--var` and `--var-file` flags.

Reply:
I forgot it. Adding it now